ui/src/app/shared/get_workflow_params.ts

```diff
@@ -0,0 +1,37 @@
+import {History} from 'history';
+/**
+ * Method to extract a key from query parameter. The query parameter will be of the following format
+ * ?parameters[key]=value.
+ * This method will extract the key from the query parameter.
+ * @param inputString
+ * @returns
+ */
+function extractKey(inputString: string): string | null {
+    // Use regular expression to match the key within square brackets
+    const match = inputString.match(/parameters\[(.*?)\]/);
```

Comment:
Probably should anchor this so it doesn't match something like `redirectparameters[foo]`
```suggestion
    const match = inputString.match(/^parameters\[(.*?)\]$/);
```